apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html

```diff
@@ -71,962 +46,1197 @@
+                            </form> -->
+
+                            <p class="mr-b15 font-14">
+                                <span>
+                                    <mat-checkbox color="primary" id="stock">
+                                        Continue selling when out of stock?
+                                    </mat-checkbox>
+                                </span>
+                            </p>
+                            <p class="mr-b15 font-14">
+                                <span>
+                                    <mat-checkbox color="primary" name="return-policy" id="return-policy">
+                                        Is this stock eligible for return policy?
+                                    </mat-checkbox>
+                                </span>
+                            </p>
+                            <p class="mr-b15 font-14">
+                                <span>
+                                    <mat-checkbox color="primary" [checked]="true" id="variant">
+                                        Variant? This product has multiple options like size or colours etc..
+                                    </mat-checkbox>
+                                </span>
+                            </p>
+                            <!-- this div wil be visible if uper checkbox checked -->
+                            <div class="product-option">
+                                <div class="product-option-wrapper">
+                                    <form>
+                                        <div class="row">
+                                            <div class="col-sm-12 col-md-6">
+                                                <div class="row mr-b15">
+                                                    <div class="col-sm-6 col-lg-4">
+                                                        <label>Option 1</label>
+                                                        <text-field
+                                                            [cssClass]="'form-control'"
+                                                            placeholder="Size"
+                                                            id="select-option"
+                                                        ></text-field>
+                                                    </div>
+                                                    <div class="col-sm-6 col-lg-6">
+                                                        <label>Category</label>
+                                                        <div class="multi-select-input">
+                                                            <mat-chip-list #chipList aria-label="Fruit selection">
+                                                                <mat-chip
+                                                                    class="s-chip"
+                                                                    *ngFor="let categorys of category"
```

Comment:
Consider using plural array name
```suggestion
                                                                    *ngFor="let category of categories"
```